src/tests/system/tests/test_schema.py

```diff
@@ -71,44 +74,43 @@ def test_schema__ldap_extra_attrs_check_ldb(client: Client, provider: GenericPro
     client.sssd.start()
 
     result = client.tools.getent.passwd("user1")
-    assert result is not None, "getent passwd user1 failed"
+    assert result is not None, "User 'user1' not found!"
 
     search = client.ldb.search(
         f"/var/lib/sss/db/cache_{client.sssd.default_domain}.ldb", f"cn=users,cn={client.sssd.default_domain},cn=sysdb"
     )
 
     user_dict = search["name=user1@test,cn=users,cn=test,cn=sysdb"]
-    assert user_dict["description"] == ["gecos user1"], "attribute 'description' was not correct"
-    assert user_dict["shell"] == ["/bin/sh"], "attribute 'shell' was not correct"
-    assert user_dict["userID"] == ["100110"], "attribute 'userID' was not correct"
-    assert user_dict["groupID"] == ["111111"], "attribute 'groupID' was not correct"
+    assert user_dict["description"] == ["gecos user1"], "Attribute 'description' was not correct!"
+    assert user_dict["shell"] == ["/bin/sh"], "Attribute 'shell' was not correct!"
+    assert user_dict["userID"] == ["100110"], "Attribute 'userID' was not correct!"
+    assert user_dict["groupID"] == ["111111"], "Attribute 'groupID' was not correct!"
 
 
 @pytest.mark.topology(KnownTopologyGroup.AnyProvider)
-def test_schema__ldap_extra_attrs_negative_cache(client: Client, provider: GenericProvider):
+def test_schema__user_extra_attributes_with_null_values_are_not_cached_or_returned(
```

Comment:
Similarly, only the returned results should be tested here but not the cache contents.